java/vertx4web/src/main/java/benchmark/vertx4web/BenchmarkApplication.java

```diff
@@ -0,0 +1,22 @@
+package benchmark.vertx4web;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpServerOptions;
+import io.vertx.ext.web.Router;
+
+public class BenchmarkApplication {
+    public static void main( String [] args )
+    {
+        Vertx vertx = Vertx.vertx();
+
+        Router router = Router.router(vertx);
+
+        router.get("/").respond( ctx -> ctx.response().setStatusCode(200).end());
+        router.post("/user").respond( ctx -> ctx.response().setStatusCode(200).end());
+        router.get("/user/:userId").respond( ctx -> ctx.response().setStatusCode(200).end(ctx.pathParam("userId")));
```

Comment:
could this be chained (I think it's clearer)
```suggestion
        router.
          get("/").respond( ctx -> ctx.response().setStatusCode(200).end())
          post("/user").respond( ctx -> ctx.response().setStatusCode(200).end())
          get("/user/:userId").respond( ctx -> ctx.response().setStatusCode(200).end(ctx.pathParam("userId")));
```

Reply:
It cannot, the API is not chainable: https://vertx.io/docs/apidocs/io/vertx/ext/web/Router.html#get--